src/pointwise_loglikelihoods.jl

```diff
@@ -0,0 +1,40 @@
+function _pd_diag_inv(A::PDMats.AbstractPDMat)
+    T = typeof(float(oneunit(eltype(A))))
+    I = Diagonal(ones(T, axes(A, 1)))
+    return PDMats.invquad(A, I)
+end
+
+function pointwise_loglikelihood!(
```

Comment:
Since it's still computing multiple likelihoods, maybe the plural name makes more sense:
```suggestion
function pointwise_loglikelihoods!(
```